client/tests/integration/smartcontracts/executor_custom_data_model/Cargo.toml

```diff
@@ -8,6 +8,9 @@ authors.workspace = true
 license.workspace = true
 
 [dependencies]
+# TODO: Cargo complains if I take it from workspace
+iroha_executor = { version = "=2.0.0-pre-rc.21", path = "../../../../../smart_contract/executor", features = ["debug"] }
```

Comment:
idk what's happening here

Reply:
ah yeah, it depends on which workspace we're referring to. It is in workspace when I compile `client/test/integration/smartcontracts` but not when I compile `client/tests`. It's a dependency in both